bitbybit-tests/tests/ui/multiple_catchall.rs

```diff
@@ -0,0 +1,14 @@
+use bitbybit::bitenum;
+
+#[bitenum(u8, exhaustive: true)]
```

Comment:
awesome tests!

I'm assuming catchall implies exhaustive? It would be useful to have a test for that as well

Reply:
Yes, that's right. Will do.